trunk/Solution/SolutionExplorer.cs

```diff
@@ -751,16 +751,21 @@ private string GetXml(int intDepth, TreeNode treeNode)
 				sb.Append('\t');
 			string strActive = "";
 			string strDescription = "";
+			string strEnd = "";
 			if (treeNode == ActiveObject)
 				strActive = @" active=""true""";
 			if (realTag.Description != string.Empty)
 				strDescription = @" description=""" + realTag.Description + @"""";
-			sb.AppendFormat("<{0} name=\"{1}\" guid=\"{2}\"{3}{4}>\r\n", strTagName, strValue, realTag.Guid, strActive, strDescription);
-			foreach (TreeNode childNode in treeNode.Nodes)
-				sb.Append(GetXml(intDepth + 1, childNode));
-			for (int intI = 0; intI < intDepth; intI++)
-				sb.Append('\t');
-			sb.AppendFormat("</{0}>\r\n", strTagName);
+			if (treeNode.Nodes.Count == 0)
+				strEnd = " /";

```

Comment:
For new code, please use braces for single line blocks. Fixing the older lines is optional unless they are changed.

Reply:
Ok, I'll add braces for single line blocks. The funny thing is that my editor forces braces for single line blocks as well "this." for locals, "var", space as tab and new line braces upon saving the code.
So, to not make a mess and keep the code format that is used across the project, I'm using Notepad to edit the code. Therefore this is the old code, I just added an "if" above.
btw, can I format all the code to my editor style? It looks pretty much as the default brand new C# style.